docker/cardano/README.md

```diff
@@ -0,0 +1,23 @@
+# Cardano docker scripts
+
+## Basic Testnet
+
+This creates a testnet with three nodes and a user account with all the funds.
+It also runs a custom version of cardano-wallet server, accessible via port 8190,
+which includes the features needed by the force-bridge collector and
+verifier operators.
+
+```
```

Comment:
Does it lack `docker build -f Dockerfile.nix-base -t  bridge_nix-base .` here?
Since when I run this, I got the error below:

```
$ docker-compose build
Building cardano-nodes-and-cli
[+] Building 3.4s (4/4) FINISHED
 => [internal] load build definition from Dockerfile.cardano-node                                                                                                   0.1s
 => => transferring dockerfile: 769B                                                                                                                                0.0s
 => [internal] load .dockerignore                                                                                                                                   0.0s
 => => transferring context: 2B                                                                                                                                     0.0s
 => ERROR [internal] load metadata for docker.io/library/bridge_nix-base:latest                                                                                     3.2s
 => [auth] library/bridge_nix-base:pull token for registry-1.docker.io                                                                                              0.0s
------
 > [internal] load metadata for docker.io/library/bridge_nix-base:latest:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message:
 insufficient_scope: authorization failed
ERROR: Service 'cardano-nodes-and-cli' failed to build : Build failed
```

Reply:
Yes, this was needed. but now I have removed the need for this step.